lib/manager/components/version/FeedVersionSpanChart.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+
+import React, { Component } from 'react'
+import { Table } from 'react-bootstrap'
+import moment from 'moment'
+
+import type { FeedVersion } from '../../../types'
+
+type Props = {
+  activeVersion: FeedVersion,
+  comparedVersion: FeedVersion,
+  comparedVersionIndex: number
+}
+
+const dateFormat = 'MMM. DD, YYYY'
+
+/** Renders the validity span of one feed using the given props. */
+const FeedSpan = ({
+  endLabel,
+  relativeLength,
+  relativeOffset,
+  startLabel,
+  widthPixels,
+  ...props
+}) => {
+  // Date labels and validity span occupy a third of the width available.
+  const baseWidth = widthPixels / 3
+  return (
+    <div {...props} style={{ position: 'relative', width: `${widthPixels}px` }}>
+      <div
+        style={{
+          left: `${relativeOffset * baseWidth}px`,
+          position: 'absolute'
+        }}>
+        <span
+          style={{ display: 'inline-block', textAlign: 'right', width: `${baseWidth}px` }}>{startLabel}</span>
```

Comment:
This is kind of hard to read. How about:

```js
<span style={{ display: 'inline-block', textAlign: 'right', width: `${baseWidth}px` }}>
  {startLabel}
</span>
```

Reply:
Updated in https://github.com/ibi-group/datatools-ui/pull/528/commits/08c881ccaf552b503dddbb05ada677e37e3a9272.